OneLogin.xcodeproj/project.pbxproj

```diff
@@ -1823,7 +1859,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/govuk-one-login/mobile-ios-common.git";
 			requirement = {
-				branch = main;
+				branch = "DCMAW-7797-passcode-information-screen";
```

Comment:
This needs to be removed once the PR Common is merged (kept it in to see if checks pass).